eng/scripts/Logging-Functions.ps1

```diff
@@ -15,7 +15,7 @@ function LogWarning {
     Write-Host ("##vso[task.LogIssue type=warning;]$args" -replace "`n", "%0D%0A")
   }
   elseif (Test-SupportsGitHubLogging) {
-    Write-Host ("::warning::$args" -replace "`n", "%0D%0A")
+    Write-Warning ("::warning::$args" -replace "`n", "%0D%0A")
```

Comment:
What does this give us? Doesn't GH just interpret these logging messages and not display them?

Reply:
At least for GH, it seemed more correct to use **both** the intended pwsh `Write-*` method, along with including the GH-specific logging strings.

`Write-Warning` might write to stdout instead of stderr, so it might not matter in practice.  What did matter in practice was using `Write-Error` instead of `Write-Host`, to ensure errors are always written to stderr.